### ISSUE TITLE
CSS is clean and succinct 

### ISSUE THREAD

**Issue by ad3429** [ShawnManek/prj-rev-bwfs-dasmoto#3]
https://github.com/ShawnManek/prj-rev-bwfs-dasmoto/blob/master/DasmotoArtsCrafts/resources/css/index.css#L6-L30

Your CSS is short, clean and succinct -- great job! One area of improvement here is to comment your main stylings so that when you come back to look at your code you know exactly why you chose to style something a particular way.

**Comment by ShawnManek** [comment]
Thanks!.

Question: If I had downloaded the background image and linked it to a root directory in CSS, would the images have to be saved in a folder within the CSS folder, so it would like to "./resources/css/images/background-image.jpeg"? OR could i save it in an images folder directly within the root directory (outside of the resources folder) and link to it within the CSS style.css file with the following image source work - "./images/background-image.jpeg"?

Basically wondering if images linked within the CSS stylesheet must link to a folder within its same folder, or does it default to the root directory with index.html and then open up the images folder there.

Thanks again for the help! Hope my question made sense.

-Shawn

**Comment by ad3429** [comment]
Hey Shawn,

great question! Either way works just fine, but if you want to exit the current directory you would have to add **../** in front. The two periods indicate to move from the the current directory (resources, for example) and go to the folder above it.

**Comment by ShawnManek** [comment]
I see. Thanks for the help and super quick response!

**Comment by ad3429** [comment]
No problem -- great job overall!
